bugreportutils/bugreportutils.go

```diff
@@ -41,80 +41,93 @@ const (
 )
 
 var (
-	// BugReportSectionRE is a regular expression to match the beginning of a bug report section.
+	// BugReportSectionRE is a regular expression to match the beginning of a
+	// bug report section.
 	BugReportSectionRE = regexp.MustCompile(`------\s+(?P<section>.*)\s+-----`)
 
 	// deviceIDRE is a regular expression that matches the "DeviceID" line.
 	deviceIDRE = regexp.MustCompile("DeviceID: (?P<deviceID>[0-9]+)")
 
-	// sdkVersionRE is a regular expression that finds sdk version in the System Properties section
-	// of a bug report.
-	sdkVersionRE = regexp.MustCompile(`\[ro.build.version.sdk\]:\s+\[(?P<sdkVersion>\d+)\]`)
+	// sdkVersionRE is a regular expression that finds sdk version in the
+	// System Properties section of a bug report.
+	sdkVersionRE = regexp.MustCompile(`\[ro.build.version.sdk\]:` +
+		`\s+\[(?P<sdkVersion>\d+)\]`)
 
-	// buildFingerprintRE is a regular expression to match any build fingerprint line in the bugreport.
-	buildFingerprintRE = regexp.MustCompile(`Build\s+fingerprint:\s+'(?P<build>\S+)'`)
+	// buildFingerprintRE is a regular expression to match any build
+	// fingerprint line in the bugreport.
+	buildFingerprintRE = regexp.MustCompile(`Build\s+fingerprint:\s+` +
+		`'(?P<build>\S+)'`)
 
-	// modelNameRE is a regular expression that finds the model name line in the System Properties
-	// section of a bug report.
-	modelNameRE = regexp.MustCompile(`\[ro.product.model\]:\s+\[(?P<modelName>.*)\]`)
+	// modelNameRE is a regular expression that finds the model name line
+	// in the System Properties section of a bug report.
+	modelNameRE = regexp.MustCompile(`\[ro.product.model\]:\s+` +
+		`\[(?P<modelName>.*)\]`)
 
 	// pidRE is a regular expression to match PID to app name and UID.
-	pidRE = regexp.MustCompile(`PID #` + `(?P<pid>\d+)` + `: ProcessRecord[^:]+:` +
-		`(?P<app>[^/]+)` + `/` + `(?P<uid>.*)` + `}`)
+	pidRE = regexp.MustCompile(`PID #` + `(?P<pid>\d+)` +
```

Comment:
This seems like an unnecessary change. Can you double check the length of the line.

Reply:
Locally I have tab size = 4, the old line exceeds the 80-character limit by just 1 character, so I made this change.

Reply:
Default at google is using 2. If you want you can skip this change